awx_collection/test/awx/test_instance_group.py

```diff
@@ -5,7 +5,6 @@
 import pytest
 
 from awx.main.models import InstanceGroup, Instance
-from awx.main.tests.functional.conftest import kube_credential, credentialtype_kube
```

Comment:
We still need this line, but can take out `credentialtype_kube`

Reply:
We don't need `kube_credential` either do we? It's passed as a fixture. Or do we need to import so that pytest loads the `conftest.py`? That doesn't make total sense to me. I think mucking with the pytest config might be the right way to fix it.

Reply:
Yes, I think we need it for the loading of `conftest.py`; your idea of changing the pytest config sounds like a good way to possibly fix this.

Reply:
I tried a little bit, but it's going to be a good deal more work to change the general pattern, and we have similar fixtures so I added this along with them.